src/adagrams.js

```diff
@@ -1,8 +1,140 @@
-const Adagrams = {
-  drawLetters() {
-    // Implement this method for wave 1
-  },
+class Adagrams {
+  static drawLetters() {
+    const letterValues = {
```

Comment:
Nice work storing the letter frequencies and then building the pool on the fly - this is much easier to read than a giant array full of 9 'A's, 2 'B's, etc.